dockers/docker-lldp-sv2/lldpd.conf.j2

```diff
@@ -2,11 +2,13 @@
 {# If MGMT port alias is available, use it for port ID subtype, otherwise use port name #}
 {% set mgmt_port_name = MGMT_INTERFACE.keys()[0][0] %}
 {% set ipv4 = MGMT_INTERFACE.keys()[0][1].split('/')[0] %}
+configure system ip management pattern {{ ipv4 }}
+{% if DEVICE_METADATA['localhost']['sub_role'] is not defined or DEVICE_METADATA['localhost']['sub_role']|length == 0 %}
```

Comment:
What is the intention here? Should we configure frontal nodes?

Reply:
@pavel-shirshov Currently we are setting portidsubtype for eth0 using (configure ports eth0 lldp portidsubtype) . Above change is to skip for lldp container that do not use eth0.  Do you see any issue here ?

Reply:
@pavel-shirshov @lguohan 
Addressed as there is no more changes in lldpd.conf.j2